src/Samples/Directory.Build.targets

```diff
@@ -1,5 +1,9 @@
 ﻿<Project>
-  
+
+  <ItemGroup>
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.23" Condition="'$(DisableNerdbankVersioning)' != 'true'" />
```

Comment:
Honestly not sure we need this since we don't ship them, but it's not terrible.

Reply:
We kinda do. We have targets elsewhere that run as after targets for GitVersioning. So we'd have to refactor that for these to build without it. Easy enough but might as well just version them the same.